src/inject/dynamic-theme/css-rules.ts

```diff
@@ -7,7 +7,14 @@ export function iterateCSSRules(rules: CSSRuleList, iterate: (rule: CSSStyleRule
     forEach(rules, (rule) => {
         if (rule instanceof CSSMediaRule) {
             const media = Array.from(rule.media);
-            if (media.includes('screen') || media.includes('all') || !(media.includes('print') || media.includes('speech'))) {
+            media.map((mediaRule) => {
```

Comment:
This `.map()` looks strange, it doesn't actually return anything. `map()` should be used to return another array, and it is not recommended to push or change anything in outer scope, because it is hard to understand.